.ci-cd/Dockerfile.cpu

```diff
@@ -32,4 +24,19 @@ RUN pip3 install -i https://mirrors.aliyun.com/pypi/simple/ \
     pylint==2.3.1 \
     yapf==0.28.0
 
-RUN apt install -y  git clang-format
+RUN apt update && apt install -y  git clang-format
+
+RUN apt install -y \
+        libsm6  \
+        libxext-dev \
+        libxrender1 \
+        xvfb
+
+# clean
+RUN  rm -rf /var/lib/apt/lists/*
+
+# python libs requirement by alf
+COPY requirements.txt  /tmp/requirements.txt
+RUN pip install -i https://mirrors.aliyun.com/pypi/simple/ -r /tmp/requirements.txt
```

Comment:
We do we need to use aliyun?

Reply:
it's faster than the default mirror source when install python packages by pip